src/com/Arrays/Task1.java

```diff
@@ -0,0 +1,23 @@
+package com.Arrays;
+
+//Создайте массив из всех чётных чисел от 2 до 20 и выведите элементы
+//        массива на экран сначала в строку, отделяя один элемент от другого пробелом, а
+//        затем в столбик (отделяя один элемент от другого началом новой строки). Перед
+//        созданием массива подумайте, какого он будет размера.
+public class Task1 {
+    public static void main(String[] args) {
+        int number = 20;
+        int length = number / 2; //находим длину массива
+        int[] mass = new int[length];
+        mass[0] = 2;
```

Comment:
why you determine this value?

Reply:
In the beginning i wanted to get mass[i]=mass[0]+i*2, but change my mind. 
Deleted mass[0]=2.